maven-plugin-plugin/src/it/beanshell-simple/pom.xml

```diff
@@ -53,6 +53,7 @@
         <artifactId>maven-plugin-plugin</artifactId>
         <version>${pluginPluginVersion}</version>
         <configuration>
+          <requiredMavenVersion>3.6.5</requiredMavenVersion>
```

Comment:
Maven version 3.6.5 is not exist ... last in 3.6.x is 3.6.3

Reply:
Fixed